components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -191,6 +191,9 @@ - (void)commonMDCBottomNavigationBarInit {
                                      UIViewAutoresizingFlexibleRightMargin);
   _containerView.clipsToBounds = YES;
   [self addSubview:_containerView];
+#if defined(__IPHONE_10_0) && (__IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_10_0)
+  _containerView.accessibilityTraits = UIAccessibilityTraitTabBar;
```

Comment:
I think we should allow this to be used on iOS 10+ today instead of waiting to drop iOS 9 support in Xcode.   Could we switch to something like this?

```objc
#if defined(__IPHONE_10_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0)
if (&UIAccessibilityTraitTabBar != NULL) {
  _containerView.accessibilityTraits = UIAccessibilityTraitTabBar;
}
#endif
```

Reply:
@romoore I didn't see this comment and just changed it to use `if @available(iOS 10.0, *)` instead. Is that alright?

Reply:
Ah, I see why this doesn't work now. I'll use the NULL check approach, thanks @romoore !